### ISSUE TITLE
Create code examples for different tasks

### ISSUE THREAD

**Issue by dvadym** [OpenMined/PipelineDP#282]
# Context

Terminology can be found [here](https://pipelinedp.io/key-definitions/).

There are 2 APIs:
1. **Core APIs** - which represent by public functions of DPEngine ([example of usage](https://github.com/OpenMined/PipelineDP/blob/main/examples/movie_view_ratings/run_all_frameworks.py))
3. **High-level APIs**: `PrivatePCollection` (Beam), `PrivateRDD` (Spark) are private counter-part of corresponding Beam/Spark objects.
They have nice guarantee: only data that has been aggregated in a DP manner, using no more than the specified privacy budget, can be extracted ([Beam example](https://github.com/OpenMined/PipelineDP/blob/main/examples/movie_view_ratings/run_on_beam.py), [Spark example](https://github.com/OpenMined/PipelineDP/blob/main/examples/movie_view_ratings/run_on_spark.py)).

It would be nice to have examples for all APIs.

# Goals
There are several [end-to-end examples](https://github.com/OpenMined/PipelineDP/tree/main/examples) on Github. The goal of this issue is to create examples with solving different tasks which PipelineDP user might have. Those examples might be snippets (not neccessary end-to-end). We will put them on [pipelinedp.io](https://pipelinedp.io/) on snippets web-page (to be created).

The initial list of those examples:
1. How to compute 1 metric DP Count.
2. How to compute 1 metric DP Sum.
3. How to compute multiple metrics per at one aggregation.
4. Using `public_partitions`.
5. Computing histogram with [VECTOR_SUM](https://github.com/OpenMined/PipelineDP/blob/e4ade1d9bd374d221ad2b3936eb33cb9e1bc943d/pipeline_dp/aggregate_params.py#L29) aggregation and 1-hot encoding.
6. Computing DP aggregations for different values (i.e. by calling several times [DPEngine.aggregate](https://github.com/OpenMined/PipelineDP/blob/e4ade1d9bd374d221ad2b3936eb33cb9e1bc943d/pipeline_dp/dp_engine.py#L53)).

**Comment by dvadym** [comment]
Thank you!

**Comment by system-reboot** [comment]
@dvadym is this issue still open? I would love to contribute towards it.

**Comment by dvadym** [comment]
@system-reboot yes this issue is open and it will be useful to have more examples. You're welcome to contribute.

You can start with a simple example with COUNT. Please let me know if you have any questions.
